OxfordBodleian/Juel-Jensen/BDLaethg34.xml

```diff
@@ -254,13 +253,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                             <additions>
                                 <list>
-                                    <item xml:id="e2">
-                                        <desc>The name of the original owner has been erased everywhere, and that of <persName ref="PRS13624TaklaG"/> been
-                                        inserted (e.g. <locus target="#1r #6v #8r"/>).</desc>
+                                        <item xml:id="e2">
+                                        <desc>Scribal correction above the line on <locus target="#6v"/>.</desc>
```

Comment:
I checked the manuscript and would suggest "An omitted letter has been added above the line on…"